public/scripts/stackedbar-util.js

```diff
@@ -1,15 +1,83 @@
 ////////////////////////////// For running time stacked bar ///////////////////
-var StackedBarUtil = (function($) {
-  var sbutil = {};
+var StackedBarUtil = (function($, StateUtil) {
```

Comment:
@initialcontext I feel like this should be renamed now, but I couldn't come up with a good name.